### PR TITLE
Hikmet-Database-Week 3

### PR DESCRIPTION
**Pull request by hikmetdag**

### CODE REVIEW

**Comment by martijnboers** [comment]
Hey @hikmetdag, next time could you only commit to the MR the code for that specific week? I can look through the folders but it's also an exercise for you to improve at your git skills

File: Week3/homework/Exercise-1.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+1-What columns violate 1NF?
+
+- food_code and food_description columns violate this rule, because every column should only contain a single value
+
+- There is no primary keys.
+
+- dinner_date column data are not the same type.
```
Comment (martijnboers) [inline Week3/homework/Exercise-1.txt:7]: very good!

File: Week3/homework/transactions/transactions-create-tables.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const createTables=[
+    `CREATE TABLE IF NOT EXIST account(
```
Comment (martijnboers) [inline Week3/homework/transactions/transactions-create-tables.js:2]: This does not run, please verify the code before you send it in for review.

File: Week3/homework/transactions/transactions-create-tables.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    account_number INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    balance FLOAT);`,
+    
+    `CREATE TABLE IF NOT EXIST account_changes(
```
Comment (martijnboers) [inline Week3/homework/transactions/transactions-create-tables.js:6]: Same here

Comment (martijnboers) [reply Week3/homework/transactions/transactions-create-tables.js:6]: Even after fixing this it doesn't work. Did this run for you before?

**Comment by hikmetdag** [comment]
I couldn't complete all my homework unfortunately, I will finish soon.

**Pull request closed without merging**